charts/rstudio-workbench/values.yaml

```
@@ -234,6 +234,8 @@ pod:
   volumeMounts: []
   # -- podAnnotations is a map of keys / values that will be added as annotations to the rstudio-pm pods
```

```suggestion
  # -- pod.annotations is a map of keys / values that will be added as annotations to the rstudio-pm pods
```
holdover from a prior age 😅